workers/ide_callback/tasks.py

```
@@ -504,3 +506,162 @@ def ide_prompt_error(
+                _emit_websocket(
+                    ps_api,
+                    room=ws_room,
+                    event=ws_event,
+                    data={
+                        "file_id": file_id,
+                        "status": "ERROR",
+                        "error": str(e)[:500],
+                    },
+                )
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)